Formula/difftastic.rb

```diff
@@ -0,0 +1,24 @@
+class Difftastic < Formula
+  desc "Diff that understands syntax"
+  homepage "https://github.com/Wilfred/difftastic"
+  url "https://github.com/Wilfred/difftastic/archive/refs/tags/0.24.0.tar.gz"
+  sha256 "195bc08b36132b4fea90942243dece209573f6123bd8c60dabaad869e7441489"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on :macos
```

Comment:
This doesn't look right, since Linux distros also package this.

Reply:
There is a failure during the build on linux which I'm not going to waste time to figure out: is it because of CI issue due to outdated compiler, or something else.

Reply:
You can add
```rb
  on_linux do
    depends_on "gcc"
  end

  fails_with gcc: "5"
```

Reply:
@SMillerDev thanks, pushed the change.